lib/CL/devices/cpu_dbk/dbk_khr_jpeg_cpu.h

```diff
@@ -0,0 +1,95 @@
+/* dbk_khr_jpeg_cpu.h - jpeg defined builtin kernel functions for cpu devices
+
+   Copyright (c) 2024 PoCL developers
+
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to
+   deal in the Software without restriction, including without limitation the
+   rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+   sell copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+   IN THE SOFTWARE.
+*/
+
+#ifndef _DBK_KHR_JPEG_CPU_H_
+#define _DBK_KHR_JPEG_CPU_H_
+
+#include "pocl_cl.h"
+
+/**
+ * @brief Do jpeg compression from a rgb image in the first argument and store
+ * the result and size in the second two arguments.
+ * @param program [in] used to retrieve device data
+ * @param kernel [out] the jpeg state is retrieved from the kernel data.
+ * @param meta [in] used for retrieving dbk attributes.
+ * @param dev_i [in] used to retrieve device data.
+ * @param arguments [in/out] pointers to data to process and store.
+ */
+POCL_EXPORT int
+pocl_cpu_execute_dbk_khr_jpeg_encode (cl_program program,
+                                      cl_kernel kernel,
+                                      pocl_kernel_metadata_t *meta,
+                                      cl_uint dev_i,
+                                      struct pocl_argument *arguments);
+
+/**
+ * @brief Create and init the jpeg state, including a turbo jpeg handle.
```

Comment:
Drop the @brief. LLVM doesn't like them and for a good reason, they are just extra as Doxygen can parse the brief without (we have used without brief since the beginning in TCE/OpenASIP).